### DISCUSSION TITLE
atproto availability within conda-forge

### DISCUSSION THREAD

**Discussion by akrherz** [MarshalX/atproto#460] [Show and tell]
FWIW, I have packaged `atproto` for [conda-forge](https://conda-forge.org/).  You can find the [repo atproto-feedstock](https://github.com/conda-forge/atproto-feedstock) with some more details on installation.  I am happy to maintain the recipe there, so am not soliciting any help from this community, but would be happy to add anybody that wishes to join in!  Thanks for the efforts on this library :partying_face:

**Comment by MarshalX** [comment]
thank you!
